docs/source/howto/run_containerized_codes.rst

```diff
@@ -0,0 +1,151 @@
+.. _how-to:run-containerized_codes:
+
+****************************************
+How to setup and run containerized codes
+****************************************
+
+This how-to walks you through the steps of setting up a compute resource, setting up containerized codes on that computer, and submitting a calculation through AiiDA (with more details supplement to :ref:`introductory tutorial <tutorial:basic:calcjob>`).
+
+.. important::
+
+    Running code in the container usually require mapping the current directory (where the input files locate) to the working directory in the container.
+    Current directory is specified in the job script as ``$PWD``.
+    The computer setup to run containerized code **MUST** set ``use_double_quotes`` to ``True`` to escape the command line parameters using double quotes.
+
+There are two type of codes the ``installed`` and ``portable``, which are all supported to be run in the container.
```

Comment:
```suggestion
There are two ways that a containerized code can be configured in AiiDA:

* The container, which already contains the code to be run, is pre-installed on a specific target computer (``installed``)
* The container provides an execution environment into which a particular code (executable and files) is copied before it is executed (``portable``)

These two types are referred to as ``installed`` and ``portable``, because an ``installed`` container is already installed on a target computer and so it will be tied to a specific :class:`~aiida.orm.computers.Computer` configured in AiiDA, whereas the ``portable`` code is not yet installed on any computer but can be retrieved from a container index (such as Docker Hub) and downloaded and run into any computer.
The ``portable`` code therefore does not require defining any particular :class:`~aiida.orm.computers.Computer` but can be sent to any configured computer when at calculation launch time.
These two concepts are implemented by the :class:`aiida.orm.nodes.data.code.containerized.InstalledContainerizedCode` and :class:`aiida.orm.nodes.data.code.containerized.PortableContainerizedCode` data plugins.
They are analogs to the plugins for non-containerized codes :ref:`InstalledCode <topics:data_types:core:code:installed>` and :ref:`PortableCode <topics:data_types:core:code:portable>`.

```